### PR TITLE
docs(beg-guide): correct usage of trans lvl in descent.md

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary
Brought up by tracernz in #docs channel on Discord that the descent page needs correction when planning arrivals on the PERF APPR page.

- Switches copy to reflect trans level
- Switches graphics
- Adds admonition to quickly discuss honeywell's implementation and why trans level is the value to be used. (Under the `!!! info`)

### Location
- docs/pilots-corner/beginner-guide/descent.md

Discord username (if different from GitHub): Valastiri#8902

### CODE REVIEW

**Review by tracernz — APPROVED** [review]

File: docs/pilots-corner/beginner-guide/descent.md [hunk 1]
```diff
@@ -193,13 +198,13 @@ ACKNOWLEDGE RECEIPT OF INFORMATION K AND ADVISE AIRCRAFT
 TYPE ON FIRST CONTACT
 ```
 
-From the chart we get `TRANS ALT` and `BARO` (=MDA) or `RADIO` (=DH).
+From the chart we get `Trans level` and `BARO` (=MDA) or `RADIO` (=DH).
```
Comment (frankkopp) [inline docs/pilots-corner/beginner-guide/descent.md]: Can we add a sentence what to do when there is no Trans Lvl? Apart from Online ATC ATIS there will prob never be a Trans Lvl in EU. What should users then enter?

**Review by frankkopp — APPROVED** [review]

File: docs/pilots-corner/beginner-guide/descent.md (hunk 1 above)
Comment (Valastiri) [reply docs/pilots-corner/beginner-guide/descent.md]: This is actually a great point - what would your suggestion be? We could say if you are not flying online with an ATC you could look up a real-world D-ATIS or for basic simulation transition at FL100?

Comment (tracernz) [reply docs/pilots-corner/beginner-guide/descent.md]: SimBrief seems to give trans alt + 1000.

Comment (Valastiri) [reply docs/pilots-corner/beginner-guide/descent.md]: That could work then. Can suggest looking both D-ATIS and as a further backup trans alt + 1000.

Comment (Valastiri) [reply docs/pilots-corner/beginner-guide/descent.md]: image

Something along these lines.

**Review by frankkopp — APPROVED** [review]

**Pull request merged**